src/tests/ftest/soak/utils.py

```diff
@@ -1134,6 +1134,64 @@ def create_fio_cmdline(self, job_spec, pool):
     return commands
 
 
+def create_app_cmdline(self, job_spec, pool, ppn, nodesperjob):
+    """Create the srun cmdline to run app.
+
+    This method will use a cmdline specified in the yaml file to
+    execute a local binary until the rpms are available
+    Args:
+        self (obj):       soak obj
+        job_spec (str):   job in yaml to run
+        pool (obj):       TestPool obj
+        ppn(int):         number of tasks to run on each node
+        nodesperjob(int): number of nodes per job
+    Returns:
+        cmd(list): list of cmdlines
+
+    """
+    commands = []
+    run_cmd = []
+    sbatch_cmds = []
+    app_params = os.path.join(os.sep, "run", job_spec, "*")
+    app_cmd = self.params.get("cmdline", app_params, default=None)
+
+    if app_cmd is None:
+        self.log.info(
+            "<<{} command line not specified in yaml; job will not be run>>".format(job_spec))
+        return
+    oclass_list = self.params.get("oclass", app_params)
+    for oclass in oclass_list:
+        add_containers(self, pool, oclass)
+        # sbatch_cmds = ["module purge", "module load {}".format(mpi_module)]
+        sbatch_cmds = ["export LD_LIBRARY_PATH=/opt/intel/oneapi/mpi/latest/lib",
+                       "export PATH=/opt/intel/oneapi/mpi/latest/bin",
+                       "/usr/bin/ls -la /opt/intel/oneapi/",
+                       "/opt/intel/oneapi/setvars.sh",
+                       "/opt/intel/oneapi/modulefiles-setup.sh --force",
+                       "echo $MODULEPATH",
+                       "module avail"]
+        # include dfuse cmdlines
+        log_name = "{}_{}_{}_{}_{}".format(
+            job_spec, oclass, nodesperjob * ppn, nodesperjob, ppn)
+        dfuse, dfuse_start_cmdlist = start_dfuse(
```

Comment:
(pylint-redundant-keyword-arg) Argument 'name' passed by position and keyword in function call
(pylint-redundant-keyword-arg) Argument 'job_spec' passed by position and keyword in function call